### PR TITLE
Add lazy objects and bcmath to the PHP 8.4 Spanish release page

### PR DESCRIPTION
**Pull request by andresayac**

### CODE REVIEW

File: releases/8.4/languages/es.php [hunk 1]
```diff
@@ -15,6 +15,8 @@
...
+    'bcmath_title' => 'API de objetos para BCMath',
+    'bcmath_description' => '<p>El nuevo objeto <code>BcMath\Number</code> permite el uso orientado a objetos y operadores matemáticos estándar cuando se trabaja con números de precisión arbitraria.</p><p>Estos objetos son inmutables e implementan la interfaz <code>Stringable</code>, por lo que se pueden usar en contextos de cadena como <code>echo $num</code>.</p>',
     'new_array_find_title' => 'Nuevas funciones <code>array_*()</code>',
     'new_array_find_description' => 'Nuevas funciones disponibles: <a href="/manual/es/function.array-find.php"><code>array_find()</code></a>, <a href="/manual/es/function.array-find-key.php"><code>array_find_key()</code></a>, <a href="/manual/es/function.array-any.php"><code>array_any()</code></a> y <a href="/manual/es/function.array-all.php"><code>array_all()</code></a>.',
     'pdo_driver_specific_subclasses_title' => 'Procesadores SQL específicos para PDO Driver',
```
Comment (sy-records) [inline releases/8.4/languages/es.php:22]: This also needs to be updated.

Comment (andresayac) [reply releases/8.4/languages/es.php:22]: Hi, I don't understand when you say that you should also update, hehe, I've already taken those changes here too.

Comment (sy-records) [reply releases/8.4/languages/es.php:22]: see https://github.com/php/web-php/commit/71c7b315b8ce6cdfa59f363b95d29488dd9532e1#diff-c4fb0bf8c8a5eeead991c1b07ee3adf1b3d888346794d481d0eda61b7309c8d2R22

Comment (andresayac) [reply releases/8.4/languages/es.php]: > see [71c7b31#diff-c4fb0bf8c8a5eeead991c1b07ee3adf1b3d888346794d481d0eda61b7309c8d2R22](https://github.com/php/web-php/commit/71c7b315b8ce6cdfa59f363b95d29488dd9532e1#diff-c4fb0bf8c8a5eeead991c1b07ee3adf1b3d888346794d481d0eda61b7309c8d2R22)

verified and ready if you are absolutely right I really appreciate the clarification

**Review by sy-records — APPROVED** [review]

**Review by sy-records — APPROVED** [review]

**Pull request merged**